rl/sb3_simple.py

```diff
@@ -113,14 +114,13 @@ def _on_step(self) -> bool:
 def objective(trial: optuna.Trial) -> float:
 
     kwargs = DEFAULT_HYPERPARAMS.copy()
-    # Sample hyperparameters
+    # Sample hyperparameters.
     kwargs.update(sample_a2c_params(trial))
-    # Create the RL model
+    # Create the RL model.
     model = A2C(**kwargs)
-    # Create env used for evaluation
-    eval_env = gym.make(ENV_ID)
-    # Create the callback that will periodically evaluate
-    # and report the performance
+    # Create env used for evaluation.
+    eval_env = Monitor(gym.make(ENV_ID))
```

Comment:
[Note] `Monitor` inherits `gym.Env`, and we don't need to update the type hints of `TrialEvalCallback`.

Reply:
I also confirmed that `Monitor` exists in v0.7.0. So, we can keep the lower version of sb3.
https://github.com/DLR-RM/stable-baselines3/blob/494ebfd20abe90acc136fdaf215c76ec566acd2c/stable_baselines3/common/monitor.py#L15

Reply:
Good to know that! Thank you for your investigation!